### DISCUSSION TITLE
Does Parcel support .browserslistrc files?

### DISCUSSION THREAD

**Discussion by cbirdsong** [parcel-bundler/parcel#7629] [Q&A] [answered]
The docs for both v1 and v2 only mention using the browserslist field in package.json. Does Parcel 2 also check for a .browserslistrc file?

**Comment by mischnic** [comment, marked as answer]
Yes, it does: https://github.com/parcel-bundler/parcel/blob/5b38bdfe68782c470e23fbc725c86a8d4a25cded/packages/core/core/src/requests/TargetRequest.js#L438

Feel free to open a PR to mention this to the docs
